wg-ecosystem/README.md

```diff
@@ -120,16 +115,28 @@ These repos are sorted alphabetically, their order does not reflect that any of
 * [`electron-userland/electron-installer-squirrel-windows`](https://github.com/electron-userland/electron-installer-squirrel-windows)
 * [`electron-userland/electron-installer-windows`](https://github.com/electron-userland/electron-installer-windows)
 * [`electron-userland/electron-installer-zip`](https://github.com/electron-userland/electron-installer-zip)
-* [`electron-userland/spectron`](https://github.com/electron/spectron)
 
 ...and all other repositories that the community would consider maintained by Electron.
 
+## Node.js Upgrade Policy
+
+As a general rule, npm packages under the purview of the Ecosystem WG will support Node.js versions as
+long as possible, even after a version has reached its End-Of-Life (EOL) date.
+If too much friction arises from the burden of maintaining EOL versions of Node.js (e.g. trouble upgrading
+dependencies or CI issues), we will perform a blanket upgrade of supported Node.js versions for all our packages
+to the latest LTS version. This upgrade process will be revisited annually.
+
+As of July 2024, our current plan is to upgrade all packages to a minimum of Node.js 22 when that version
+reaches its LTS date ([see Node.js release schedule for more details](https://nodejs.org/en/about/previous-releases#release-schedule)).
```

Comment:
I think we were also discussing jumping to 20.x if the ESM changes are backported. If we do 22 we'll probably want to wait N months after LTS rather than jumping to it right after it reaches LTS.

Reply:
Yeah maybe avoid specifying the version, say "the first LTS version with require-esm support" or something and link the node issue

Reply:
[f9c421d](https://github.com/electron/governance/pull/578/commits/f9c421d6076f93a4bd06001cf27da91f2c612745)